resfit/fit_resonator/resonator.py

```diff
@@ -301,15 +301,15 @@ def __init__(self,
                  vary: bool = None):
         self.method = method
         if method.name == 'DCM':
-            self.func = fit_functions.Cavity_DCM
+            self.func = fit_functions.cavity_DCM
         elif method.name == 'DCM_REFLECTION':
-            self.func = fit_functions.Cavity_DCM_REFLECTION
+            self.func = fit_functions.cavity_DCM_REFLECTION
         elif method.name == 'PHI':
-            self.func = fit_functions.Cavity_DCM
+            self.func = fit_functions.cavity_DCM
         elif method.name == 'INV':
-            self.func = fit_functions.Cavity_inverse
+            self.func = fit_functions.cavity_inverse
         elif method.name == 'CPZM':
-            self.func = fit_functions.Cavity_CPZM
+            self.func = fit_functions.cavity_CPZM
```

Comment:
Up to you, but you can also consider moving this into the enum:
```python
class FittingMethod(enum.Enum):
    DCM = 1
    DCM_REFLECTION = 2
    PHI = 3
    INV = 4
    CPZM = 5

    def get_function(self):
        if self == FittingMethod.DCM:
            return fit_functions.cavity_DCM
        elif self == FittingMethod.DCM_REFLECTION:
            return fit_functions.cavity_DCM_REFLECTION
        elif self == FittingMethod.PHI:
            return fit_functions.cavity_DCM
        elif self == FittingMethod.INV:
            return fit_functions.cavity_inverse
        elif self == FittingMethod.CPZM:
            return fit_functions.cavity_CPZM

        raise ValueError('Unknown fitting method: {}'.format(self))


class FitMethod(object):
    def __init__(self, ...):
        self.func = FittingMethod(method).get_function()
```

Reply:
For some reason the enum won't evaluate itself correctly. If I create
```python
fit = FittingMethod.DCM
fit.get_function()
>>>Unknown fitting method: FittingMethod.DCM
```

And if I just put `self == FittingMethod.DCM` evaluates to `false`

I'm going to punt this for a future PR